README.md

```diff
@@ -1,7 +1,9 @@
 Mirah
 ================
 
-[![Build Status](https://secure.travis-ci.org/mirah/mirah.png)](http://travis-ci.org/mirah/mirah)
+| **Tarvis** |
```

Comment:
Should this read `Travis`?

Reply:
Yes that should be Travis. I have made the changes in PR.